browser/brave_ads/ads_tooltips_delegate_impl.h

```diff
@@ -0,0 +1,40 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_BRAVE_ADS_ADS_TOOLTIPS_DELEGATE_IMPL_H_
+#define BRAVE_BROWSER_BRAVE_ADS_ADS_TOOLTIPS_DELEGATE_IMPL_H_
+
+#include <string>
+
+#include "brave/browser/brave_ads/tooltips/ads_tooltips_controller.h"
```

Comment:
I think you're also missing a dep for this, but why is the `AdsTooltipsDelegateImpl` in `browser/brave_ads` and `ads_tooltips_controller` is in `browser/brave_ads/tooltips`?

Reply:
Good point, they're all in `browser/brave_ads/tooltips` now.